src/auto_archiver/modules/gsheet_processor/__manifest__.py

```diff
@@ -0,0 +1,58 @@
+{
+    "name": "Google Sheets Procesor",
+    "type": ["feeder"],
+    "requires_setup": True,
+    "external_dependencies": {
+        "python": ["loguru", "gspread", "python-slugify"],
+    },
+    "configs": {
+        "allow_worksheets": {
+            "default": set(),
+            "help": "(CSV) only worksheets whose name is included in allow are included (overrides worksheet_block), leave empty so all are allowed",
+            "cli_set": lambda cli_val, cur_val: set(cli_val.split(","))
```

Comment:
these `cli_set` vals can be removed, they're not needed anymore. We should use the `type=XXX` key/value on these dicts now, since that's what argparse uses.